### DISCUSSION TITLE
Regarding little slow fading updates between markers on move

### DISCUSSION THREAD

**Discussion by Akash-pugazh** [patrykandpatrick/vico#646] [Questions] [answered]
I have a scenario where the graph can me listened by the user on click and move using the MarkerVisibilityChangeListener but there is a bit of slow updates not sure it was the proper working because the doc said it would be called on change of the marker, is there any way to fix this like instant updates on the custom marker and view model states. Thank you for helping me out in advance!

https://github.com/patrykandpatrick/vico/assets/89995392/1efcc869-ecdf-40a4-b89c-1f31dfb47626

**Comment by patrickmichalik** [comment, marked as answer]
Hello! The `MarkerVisibilityChangeListener` functions are called instantly. Could you please share a minimal reproducible example? Seeing that, in the recording, the `Marker` label updates with a delay—which doesn’t occur in the sample app—I believe there may be an issue with your implementation of Vico.

**Reply by Akash-pugazh** [reply]
Okey fine then I'll check out my implemeted usage, then thanks for reply man🫡

**Reply by patrickmichalik** [reply]
You’re welcome! If you have any further questions, feel free to ask.
